frontend/src/views/Sponsor.vue

```diff
@@ -0,0 +1,204 @@
+      <v-img max-width="80vw" max-height="30vh" contain src="@/assets/csesocwhiteblue.png" />
+    </header>
+    <v-container class="margin" fluid>
+      <h1 class="border text-h1 font-weight-bold" style="padding:25px;">Sponsors</h1>
+      <h2 class="text-h4">Principal <br> Sponsors</h2>
+      <v-container class="border" fluid>
+          <div v-for="(sponsor, index) in tierOne" :key="sponsor.id" :style="marginStyle(index, largeLogoFilter)">
+              <div class="logo-margin">
+                <img class="large-logo" :src="sponsor.logo" v-on:click="onClickModal(sponsor)"/>
+              </div>
+              <div v-if="(index + 1) % largeLogoFilter === 0 || index === tierOne.length - 1" style="clear:both;"></div>
+          </div>
+      </v-container>
+      <div style="clear:both;"></div>
+      <h2 class="text-h4">Major <br> Sponsors</h2>
+      <v-container class="border" fluid>
+          <div v-for="(sponsor, index) in tierTwo" :key="sponsor.id" :style="marginStyle(index, midLogoFilter)">
+              <div class="logo-margin">
+                <img class="mid-logo" :src="sponsor.logo" v-on:click="onClickModal(sponsor)"/>
+              </div>
+              <div v-if="(index + 1) % midLogoFilter === 0 || index === tierTwo.length - 1" style="clear:both;"></div>
+          </div>
+      </v-container>
+      <div style="clear:both;"></div>
+      <h2 class="text-h4">Affiliate <br> Sponsors</h2>
+      <v-container class="border" fluid>
+          <div v-for="(sponsor, index) in tierThree" :key="sponsor.id" :style="marginStyle(index, smallLogoFilter)">
+              <div class="logo-margin">
+                  <img class="small-logo" :src="sponsor.logo" v-on:click="onClickModal(sponsor)"/>
+              </div>
+              <div v-if="(index + 1) % smallLogoFilter === 0 || index === tierThree.length - 1" style="clear:both;"></div>
+          </div>
+      </v-container>
+      <div style="clear:both;"></div>
+    </v-container>
+    <SponsorModal v-bind:dialog="dialog" v-bind:title="currentSponsor.name" v-bind:body="currentSponsor.detail"> </SponsorModal>
+    
+    <h1 class="text-center text-h1 font-weight-bold">Become a <br> Sponsor</h1>
+    <v-container fluid style="margin-left:15%">
+      <label class="text-body-1 input-label"> Company Name </label>
+      <v-text-field class="input" placeholder="Company Name" v-bind:value="newCompanyName" v-model="newCompanyName"></v-text-field>
+      <label class="text-body-1 input-label" style="margin-right:150px;"> Email </label>
+      <v-text-field class="input" placeholder="John.smith@gmail.com" v-bind:value="newCompanyName" v-model="newEmail"></v-text-field>
+      <label class="text-body-1 input-label" style="margin-right:150px;"> Message </label>
+      <v-textarea class="input" placeholder="Body" v-bind:value="newCompanyName" v-model="newMessage"></v-textarea>
+      <v-btn text style="margin-left:62%" @click="onClickSend()">Send</v-btn>
+    </v-container>
```

Comment:
When the window is resized horizontally the input fields in the "Become a Sponsor" form shift horizontally as well. Seems like a CSS issue 😥

Reply:
I'm not quite sure how to fix this, taken a look into it, @glebme do you think this is a necessary issue we need to fix right now?

Reply:
Will see if i can have a go at it.

Reply:
removing the style attribute from the labels fixes the issue:
`style="margin-right:150px;"`